### PR TITLE
Change to Nextcloud

### PR DESCRIPTION
**Pull request by JSoko**

### CODE REVIEW

File: user_manual/files/file_drop.rst [hunk 1]
```diff
@@ -1,60 +1,42 @@
...
-other people can upload files to you without having to log in to the server, and 
-without being an ownCloud user. They will not be allowed to see the contents of 
-this directory, or to make any changes. This is an excellent alternative to 
-sending large attachments via email, using an FTP server, or using commercial 
-file-sharing services.
+===========================
+Making Anonymous Uploads
+===========================
+
+If your Nextcloud administrator has enabled the Files Drop application, you may
```
Comment (MorrisJobke) [inline user_manual/files/file_drop.rst]: This is no separate app anymore. It's now simply another option for share links.

File: user_manual/files/file_drop.rst [hunk 2]
```diff
@@ -1,60 +1,42 @@
...
+Nextcloud send funktion, if your administrator has enabled it.
 
-.. image:: ../images/files-drop-4.png
 
 Uploading Files
 ---------------
 
-Using the Files Drop app is simple. You receive a link to the upload 
-folder, click the link, and then you'll see an ownCloud page with a **Click to 
+Using the anonymous upload funktion ist  simple. You receive a link to the upload
```
Comment (MorrisJobke) [inline user_manual/files/file_drop.rst]: ;)

**Comment by MorrisJobke** [comment]
I squashed all the commits of the old PR into one single commit. I will now cherry-pick over the new commit in here and merge this. Maybe then fetch the latest version of the documentation repo (or go the easier way and delete your repo and fork again - if there are no additional changes that aren't merged yet).

**Comment by MorrisJobke** [comment]
Thanks a lot for your contribution :)

**Comment by MorrisJobke** [comment]
I created a branch in this repo, that contains only your newest changes: https://github.com/nextcloud/documentation/compare/JSoko-master

Either use this as base. Beside that I can also manage to help out with the git juggling. (most of the trouble here only happened, because I squashed all commits into one to not clutter the git history that much). ;)

File: user_manual/files/file_drop.rst (hunk 1 above)
Comment (JSoko) [inline user_manual/files/file_drop.rst]: Yes I know. And I have changed the text completly with new screenshots...
... but forgotten to delete the sentence about the applcation.

Changed...

**Comment by JSoko** [comment]
@MorrisJobke 

Thank you!
I have to commit, that this is a littlebit to complicated for me ;-)

I'll try my best...

Ciao,
Joachim

File: user_manual/files/file_drop.rst (hunk 2 above)
Comment (MorrisJobke) [inline user_manual/files/file_drop.rst]: Could you fix this typo here too ;)

**Comment by MorrisJobke** [comment]
> I have to commit, that this is a littlebit to complicated for me ;-)

Hey, no problem. I will fix this :) Thanks for all your work. :)

**Comment by nickvergessen** [comment]
Should get this in, want to do it @MorrisJobke or should I?

**Comment by MorrisJobke** [comment]
> Should get this in, want to do it @MorrisJobke or should I?

@nickvergessen Feel free to do it :) I have other stuff to do here.

**Comment by nickvergessen** [comment]
Done in #57

**Pull request closed without merging**